project/app/src/main/res/values-de/strings.xml

```diff
@@ -150,4 +150,6 @@
     <string name="preferencesDeviceName">Geräte ID</string>
     <string name="preferencesHost">Hostnamen</string>
     <string name="distance_label">Abstand</string>
+    <string name="preferencesDarkMode">Dunkles Design aktivieren</string>
+    <string name="preferencesDarkModeSummary">Wechselt zu einem dunklen Design</string>
```

Comment:
Here's the thing - I'd definitely say "Dak Mode aktivieren", i.e. treat that as a German word (maaaany English words have become part of spoken German in recent years).

BUT it looks like Android 10 calls its dark mode "Dunkles Design" in the German translation (see https://www.giga.de/downloads/android-p/tipps/android-dark-mode-aktivieren-so-geht-s/), so I chose that.

Reply:
I love how that article is full of "Dark Mode" and even the URL has it. ;-) 
I personally feel that "Dark Mode" makes more sense, but who am I.

Reply:
> I personally feel that "Dark Mode" makes more sense

let's use that one then, sans quotes :smile: